guides/getting-started/using-microsoft-dev-tunnels.md

```diff
@@ -0,0 +1,90 @@
+# Using dev tunnels <Badge type="warning" text="client" /><Badge type="danger" text="server" />
+
+When you are developing an Asp.NET or Asp.NET Core based web application running on your developer machine, usually you can connect to the server using the pre configured `applicationUrl` (e.g https://localhost:44359)
+
+If you are, however, integrating an Android or iOS application using MAUI or Avalonia, the app will run either on a physical device or on a local emulator/simulator. In either case using localhost is not an option.
+
+Instead you can use [Microsoft dev tunnels](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/overview).
+
+This tools provides you with a URL that you can connect to. Any traffic received will be routed to your local web application.
+
+## Microsoft dev tunnels
+
+Microsoft dev tunnels are available for [Visual Studio Code](https://code.visualstudio.com/docs/editor/port-forwarding) and [Visual Studio](https://learn.microsoft.com/en-us/aspnet/core/test/dev-tunnels?view=aspnetcore-8.0)
+
+Explaining these tools in detail is outside the scope of this guide. Please refer to the following links to
+- [Create a dev tunnel in Visual Studio Code](https://code.visualstudio.com/docs/editor/port-forwarding)
+- [Create a dev tunnel in Visual Studio 2022](https://learn.microsoft.com/en-us/aspnet/core/test/dev-tunnels?view=aspnetcore-8.0)
+
+### Configure the IssuerUrl
+
+In order to use this tunnel now, you need to set the `IssuerUrl` in the client project (e.g. `MauiProgram.cs`)
```

Comment:
Note: the property is named `Issuer`, not `IssuerUrl` (BTW, as a general rule, the more specific "URL" name is not used in OAuth 2.0/OIDC, that both prevent the more generic "URI" concept).